cmd/buildkitd/config/gcpolicy_unix.go

```diff
@@ -5,25 +5,18 @@ package config
 
 import (
 	"syscall"
-)
 
-func DetectDefaultGCCap() DiskSpace {
-	return DiskSpace{Percentage: 10}
-}
+	"github.com/moby/buildkit/util/bklog"
+)
 
-func (d DiskSpace) AsBytes(root string) int64 {
-	if d.Bytes != 0 {
-		return d.Bytes
-	}
-	if d.Percentage == 0 {
-		return 0
-	}
+var DiskSpacePercentage int64 = 20
 
+func getDiskSize(root string) int64 {
 	var st syscall.Statfs_t
 	if err := syscall.Statfs(root, &st); err != nil {
+		bklog.L.Warnf("failed to get disk size: %v", err)
 		return defaultCap
 	}
 	diskSize := int64(st.Bsize) * int64(st.Blocks)
-	avail := diskSize * d.Percentage / 100
-	return (avail/(1<<30) + 1) * 1e9 // round up
```

Comment:
nit: @tonistiigi I didn't see if this logic here was rounding up, what was the original intent?
```
// For instance, 746943672320 bytes
// 10% = 74_694_367_232
// with the rounding-up logic:
>>> 746943672320 * 0.1
74694367232.0
>>> (74694367232/(1<<30) + 1) * 1e9
70_564_550_399.78027
```

Reply:
Perhaps we could just leave everything to be in terms of GiB, MiB, KiB, ...

Reply:
With integer division it is different https://go.dev/play/p/Ne6uQKL-F4- . Although maybe it should round up to GiB, not GB.

Reply:
got it, thanks! Will retain previous behavior for now.